### ISSUE TITLE
discount_next not shown correctly with coupon

### ISSUE THREAD

**Issue by rmclain** [recurly/recurly-js#209]
When using a single use coupon, or a 1 month lifetime coupon code the discount_next is showing that a discount is being applied when in fact it's not.  

```
<span data-recurly="discount_next"></span>
```

For example a $50 coupon setup as a single use: 

Due now:
Discount: $ 50.00
Subtotal: $ 49.00
Tax: $ 0.00
Total: $ 49.00

Due later:
Discount: $ 50.00
Subtotal: $ 49.00
Tax: $ 0.00
Total: $ 49.00

```
[Log] Object
code: "50"
discount: Object
amount: Object
USD: 50
__proto__: Object
type: "dollars"
__proto__: Object
name: "50"
single_use: true
__proto__: Object

```

**Comment by rmclain** [comment]
I have created a jsFiddle which reproduces this issue.
http://jsfiddle.net/ssnb9405/1/

**Comment by snodgrass23** [comment]
A fix for this is currently being reviewed for release. See:  https://github.com/recurly/recurly-js/pull/204

**Comment by snodgrass23** [comment]
The fix had been merged into the master branch and will be present in the next release.

**Comment by rmclain** [comment]
@snodgrass23 Thank you.

**Comment by rmclain** [comment]
@snodgrass23 is there anyway to use this in production or anytime frame for being pushed out into the next release?

**Comment by snodgrass23** [comment]
You could pull down the repo and manually run the build and then serve it locally from the resulting file. I'll look into getting this released, though, so you don't need to do that.

**Comment by rmclain** [comment]
Thanks, building it seemed to work great.

**Comment by snodgrass23** [comment]
https://github.com/recurly/recurly-js/pull/213

Looks like we'll be getting the new release out next week once testing is complete.
